### PR TITLE
[fix](erc20) convert all fee values to fixed

### PR DESCRIPTION
**Pull request by dzdidi**
Signed-off-by: Denys Bitaccess <denys@bitaccess.co>

### CODE REVIEW

File: packages/ethereum-payments/src/BaseEthereumPayments.ts [hunk 1]
```diff
@@ -350,6 +350,7 @@ implements BasePayments
     const payport = await this.getPayport(unsignedTx.fromIndex)
 
     const unsignedRaw: any = cloneDeep(unsignedTx.data)
+    unsignedRaw.fee
```
Comment (hudgins) [inline packages/ethereum-payments/src/BaseEthereumPayments.ts:353]: I think this is kind of a typo?

Comment (dzdidi) [reply packages/ethereum-payments/src/BaseEthereumPayments.ts:353]: yes, thank you, this should not have been here at all

**Comment by dzdidi** [comment]
Rarely and never are two different things. When do we use them?

**Comment by dylanseago** [comment]
@dzdidi Correction, I thought they were only used when manually specified, but in fact they're used to bump fees

**Pull request merged**